contracts/ownership/Whitelist.sol

```diff
@@ -11,84 +11,88 @@ import "./rbac/RBAC.sol";
  * @dev This simplifies the implementation of "user permissions".
  */
 contract Whitelist is Ownable, RBAC {
-  event WhitelistedAddressAdded(address addr);
-  event WhitelistedAddressRemoved(address addr);
-
   string public constant ROLE_WHITELISTED = "whitelist";
 
   /**
    * @dev Throws if called by any account that's not whitelisted.
+   * @dev kept as `onlyWhitelisted` for backwards compatibility
```

Comment:
If this is deprecated, it's nice to mention the name of the modifier that replaces it.